backend/games/tests/test_leaderboard.py

```diff
@@ -0,0 +1,79 @@
+from datetime import datetime
+from games.models.scores import Score
+from backend.tests_utils import WeakAuthenticationBaseTestCase
+from django.db.models import Sum
+
+
+class LeaderboardTestCase(WeakAuthenticationBaseTestCase):
```

Comment:
Should we add a test that tries to submit a negative score?

Reply:
U are right, it should be tested 👍 I can add this test in `test_scores.py`, in another PR.